api/turing/cluster/servicebuilder/fluentd.go

```diff
@@ -77,7 +77,7 @@ func (sb *clusterSvcBuilder) NewFluentdService(
 			Volumes:               volumes,
 			VolumeMounts:          volumeMounts,
 		},
-		Replicas: fluentdReplicaCount,
+		Replicas: FluentdReplicaCount,
```

Comment:
where is the flush rate configured?

Reply:
It's configured here: https://github.com/caraml-dev/turing/blob/40df8e9bb65849f396fc394c8f60f845571f4e59/api/turing/cluster/servicebuilder/fluentd.go#L43 but I think I initially didn't multiply the interval by the number of replicas because I thought it'd be better to have it configured manually in the API configs but now I've decided that it's probably simpler to have it configured automatically. One of my newer commits should have that value multiplied by the number of replicas.